_posts/2024-08-02-new-container-names.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: New container names for puppetserver and puppetdb
+date: 2024-08-02
+github_username: rwaffen
+category: changelog
+---
+
+Beginning of today we are also releaseing our puppetserver and puppetdb containers with shorter names.
+
+The current names are: `voxpupuli/container-puppetserver` and `voxpupuli/container-puppetdb`.
+
+- <https://hub.docker.com/r/voxpupuli/container-puppetserver>
```

Comment:
the link rendering was a bit tricky in the past, I don't know if this properly renders to hyperrefs. If not we can update it to `[hub.docker.com/r/voxpupuli/container-puppetserver](https://hub.docker.com/r/voxpupuli/container-puppetserver)`

Reply:
this is official markdown, but i can change it to this longer syntax

Reply:
people in the internet say it should work this way
> try surrounding the link with angle brackets like <http://example.com>. Jekyll uses Kramdown, this is from their docs

i want to try this and see whats happen